src/Collection/access.ts

```diff
@@ -18,12 +20,15 @@ export async function hasAccess(
   eth_address: string,
   collection: CollectionAttributes
 ): Promise<boolean> {
-  const [isOwner, isCommittee] = await Promise.all([
+  const [isOwner, isCommittee, hasManagerAccess] = await Promise.all([
     new Ownable(Collection).isOwnedBy(collection.id, eth_address),
     isCommitteeMember(eth_address),
+    isTPCollection(collection)
+      ? thirdPartyAPI.isManager(collection.third_party_id, eth_address)
+      : isManager(eth_address, collection),
   ])
 
-  return isOwner || isCommittee || isManager(eth_address, collection)
+  return isOwner || isCommittee || hasManagerAccess
```

Comment:
Think so, but at the time thought that it was covered by router tests and left it alone mostly because I have to create `access.spec.ts`, still I could go that route. Wdyt?